### PR TITLE
first draft Quick Start re-write

### PR DESCRIPTION
**Pull request by rosemarymarano**
This first pass at the Quick Start Guide has the following outages:
- In Step 4 c, the command to configure Traefik to manage Ingresses created in the domain namespace.
- In Step 5. Create a domain in the domain namespace, I need the the commands or exact procedures for all the substeps:
a. Create a new image with a domain home by running the create-domain-in-image script.
b. Use kubectl apply -f on the YAML file to create the domain resource.
c. Confirm that the operator started the servers for the domain by using kubectl.
   Use kubectl to show that the domain resource was created.
   Look at the operator log.
d. Create an Ingress for the domain, where are they specifying the domainUID and domain namespace?
e. Confirm that the load balancer noticed the new Ingress and is successfully routing to the domain's server pods.
- In Step 6. Remove the domain, I need the the commands or exact procedures for all the substeps.
- In Step 7a, I need the the command or exact procedure for configuring the Traefik load balancer to stop managing the Ingresses in the domain namespace.

Thank you.

### CODE REVIEW

File: site/quickstart.md [hunk 1]
```diff
@@ -1,159 +1,171 @@
...
-          ./create-domain.sh
-          -i create-domain-inputs.yaml
-          -o /path/to/output-directory
+c.  Configure Traefik to manage Ingresses created in this namespace:
+```
+$ helm upgrade \
+  --reuse-values \
+  --set something about namespaces
+```
+d.  Configure the operator to manage the domains created in this namespace:
```
Comment (moreaut) [inline site/quickstart.md]: This is a repeat of (b)

**Comment by moreaut** [comment]
Overall comment - a lot of places say to use 'kubectl' to verify stuff.  Do we want to show the specific kubectl commands?   Are you planning to add that later?

File: site/quickstart.md [hunk 2]
```diff
@@ -1,159 +1,171 @@
...
+a.	Pull the operator image:
 ```
 $ docker pull oracle/weblogic-kubernetes-operator:2.0
 ```
-For the Traefik image:
+b.	Pull the Traefik load balancer image:
 ```
 $ docker pull traefik:latest
 ```
+c.	Pull the WebLogic 12.2.1.3 install image:
```
Comment (moreaut) [inline site/quickstart.md]: Don't we also need a step for building a patched 12.2.1.3 weblogic image?

File: site/quickstart.md (hunk 1 above)
Comment (rosemarymarano) [reply site/quickstart.md]: I definitely want to show the exact commands (for everything). That's the beauty of the quick start. Everything should be included in one place, very specific and easy to follow.

File: site/quickstart.md (hunk 2 above)
Comment (rosemarymarano) [reply site/quickstart.md]: Maybe. I'm hoping that you or Monica gives me these specifics (as I don't know them).

Comment (mriccell) [inline site/quickstart.md]: The very next step needs to patch the WLS 12.2.1.3 image.  Point them to https://github.com/oracle/docker-images/tree/master/OracleWebLogic/samples/12213-patch-wls-for-k8s

File: site/quickstart.md [hunk 3]
```diff
@@ -1,159 +1,162 @@
...
-
-* Optionally, create a configuration overrides template and any additional Kubernetes secrets it needs (for example, to override the domain home configuration of a database URL, username, and password).
+## 5. Create a domain in the domain namespace.
 
-* Create a domain resource in the domain namespace.
-  * Specify the following information: domain UID, service account, secret name, the domain home details, and optionally, the configuration overrides template name.
+a.	Create a new image with a domain home by running the [`create-domain-in-image`](../kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/README.md) script.
+* First, create a directory for the files generated by the sample.
+* Then, copy the `inputs.yaml` file from the sample and modify it:
+  * Specify a `domainUID`:
```
Comment (mriccell) [inline site/quickstart.md]: domainUID: sample-domain1

File: site/quickstart.md (hunk 1 above)
Comment (rosemarymarano) [reply site/quickstart.md]: I removed the redundant step 4d.

File: site/quickstart.md [hunk 4]
```diff
@@ -1,159 +1,162 @@
...
+    * The operator will notice and start up the servers in the domain.
 
-```
-$ helm update kubernetes/charts/weblogic-operator --name my-operator --namespace weblogic-operator-ns --values values.yaml --wait
-```
-
-* Configure the Traefik load balancer to manage the domain as follows:
-  * Create an Ingress for the domain in the domain namespace (it contains the routing rules for the domain):
+c.	Confirm that the operator started the servers for the domain by using `kubectl`.
+* Also, use `kubectl`to show that the domain resource was created.
```
Comment (mriccell) [inline site/quickstart.md]: $ kubectl describe domain sample-domain1 -n sample-domain1-ns

File: site/quickstart.md [hunk 5]
```diff
@@ -1,159 +1,162 @@
...
+    * This tells the operator to manage the new domain.
+    * The operator will notice and start up the servers in the domain.
 
-```
-$ helm update kubernetes/charts/weblogic-operator --name my-operator --namespace weblogic-operator-ns --values values.yaml --wait
-```
-
-* Configure the Traefik load balancer to manage the domain as follows:
-  * Create an Ingress for the domain in the domain namespace (it contains the routing rules for the domain):
+c.	Confirm that the operator started the servers for the domain by using `kubectl`.
```
Comment (mriccell) [inline site/quickstart.md]: $ kubectl get po -n sample-domain1-ns

File: site/quickstart.md [hunk 6]
```diff
@@ -1,159 +1,162 @@
...
 
+d.	Create an Ingress for the domain, in the domain namespace, by using the [sample](../kubernetes/samples/charts/ingress-per-domain/README.md) Helm chart:
+* Use `helm install`, specifying the `domainUID` and domain namespace.
 ```
 $ cd kubernetes/samples/charts
 $ helm install ingress-per-domain --name domain1-ingress --value values.yaml
 ```
 
-(At this point, do they have a WebLogic Kubernetes deployment in a Kubernetes cluster? If so, we have to give them something to look at, to verify their results.)
+e.	Confirm that the load balancer noticed the new Ingress and is successfully routing to the domain's server pods.
```
Comment (mriccell) [inline site/quickstart.md]: $ curl --silent http://${HOSTNAME}:30305/sample-domain1/

File: site/quickstart.md (hunk 3 above)
Comment (rosemarymarano) [reply site/quickstart.md]: Got it.

File: site/quickstart.md (hunk 4 above)
Comment (rosemarymarano) [reply site/quickstart.md]: got it, thanks.

File: site/quickstart.md (hunk 5 above)
Comment (rosemarymarano) [reply site/quickstart.md]: Got it.

File: site/quickstart.md (hunk 6 above)
Comment (rosemarymarano) [reply site/quickstart.md]: Got it. Should I remove, Browse to a URL? IS there a sample application (in the domain home in image) that they can browse to?

File: site/quickstart.md [hunk 7]
```diff
@@ -1,159 +1,177 @@
...
+* First, create a directory for the files generated by the sample.
+* Then, copy the `inputs.yaml` file from the sample and modify it:
+  * Specify a `domainUID`: `sample-domain1`
+  * Specify the domain namespace: `sample-domains-ns1`
 
+b.	The sample also creates a domain resource YAML file.
+* Also, use `kubectl apply -f` on the YAML file to create the domain resource:
 ```
-$ helm update kubernetes/charts/weblogic-operator --name my-operator --namespace weblogic-operator-ns --values values.yaml --wait
+kubectl apply -f /path/to/output-directory/weblogic-domains/sample-domain1/domain.yaml
```
Comment (doxiao) [inline site/quickstart.md]: "/path/to/output-directory" is the directory that is created in the first step. In step b. the domain resource YAML file is generated in </path/to/outpit-directory>/weblogic-domains/sample-domain1.

Comment (rosemarymarano) [reply site/quickstart.md]: I will point them to the domain-home-in-image/README.md and just make sure that they have all the specific inputs they need (like the domain UID, the domain namespace and how to "reate the Kubernetes secrets username and password of the admin account in the same Kubernetes namespace as the domain."

File: site/quickstart.md (hunk 2 above)
Comment (rosemarymarano) [reply site/quickstart.md]: I added it (from Monica).

**Pull request merged**